app_description.txt

```diff
@@ -0,0 +1,49 @@
+Zingo! A Zcash Application
+
+Use zero knowledge cryptography to manage your transaction history in a private and mindful way.
+
+Take exclusive and unique control of your financial history. Use this unique insight into your
+own financial behavior, to introspect on how you spend money. Share insights on your terms. 
+Establish healthy relationships with diverse economic communities.
+
+Only you have the power to share your information! Your perspective matters. You matter.
+
+When you send Zcash you can write a memo containing anything you like. When you receive
+Zcash you also receive the memo that the Sender wrote. As with most cryptocurrencies, wallets
+routinely send transactions to themselves to recover "change" from the transaction.  In this case, where
+the application is both the Sender and the Receiver, Zingo uses the memo system to write
+notes to itself about the transactions it builds for the User.
+
+Zingo extracts that data to provide insights exclusively to you.  In our initial release we
+already use this introspective mechanism to "remember" which addresses you've previously sent Zcash.
+In other words, Zingo keeps track of who you've sent Zcash to, on the blockchain. This "Address Book"
+of people you've "Sent To" in the past is completely private and, as long as you properly protect your
+seed-phrase, will be fully recovered even if you lose your device. 
+
+We're currently implementing another use of these on-chain memos, that will allow you to keep track (even if 
+you lose your device) of which addresses you previously produced to receive funds at. That is, you'll always
+have a full list of all the addresses you asked people to use to send you zcash.
+
+Taken together these two instrospective mechanisms provide you with insight into who you've sent
+funds to, and who you've asked for funds from.  This is just the beginning!  We'll soon implement
+mechanisms to allow users to share their insights in an anonymous, and economically viable way that
+will allow communities to aggregate around marketplaces of interesting insights!
```

Comment:
I'm concerned about this assertion. I agree that this is an idea we're interested in pursuing, but "We'll soon implement" is stronger of a promise than I would like to make.

Reply:
Switched to "we intend to"